docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
```

Comment:
Ensure that the volume name `my_media` is correctly spelled and consistent throughout your configuration. Previously, there was a typo `my_meda` mentioned, so double-check to ensure consistency.